### ISSUE TITLE
Classification performance of b4,b5,b6,b7 is worse than b3,Is it right?

### ISSUE THREAD

**Issue by wgqbit95** [lukemelas/EfficientNet-PyTorch#251]
When I set the input image at a fixed resolution of 256*256, the performance of b3 is better than others. I wonder whether it is the correct condition?

**Comment by Surayuth** [comment]
Have you checked the training accuracy? May be the models overfitted the dataset.

**Comment by lukemelas** [comment]
It's possible that this is the case if you're not using the resolutions that B4/5/6/7 were trained on.
